js/tests/unit/carousel.js

```diff
@@ -1,6 +1,8 @@
 $(function () {
   'use strict'
 
+  var touchSupported = 'ontouchstart' in document.documentElement
+
```

Comment:
for testing purposes, this is of course fine. i'll just note here that to detect pointer events touch support you'd want to also check for `navigator.maxTouchPoints > 0` (in production on a live site, not for automated unit tests)

Reply:
yep, I added that line because on Safari our unit tests with touch events failed 😄